src/components/UriViewer/UriDownloadButton.js

```diff
@@ -55,16 +57,18 @@ export const UriDownloadButton = ({ uri, metadata: { bucket, name, fileName, siz
   useOnMount(() => {
     getUrl()
   })
+
+  const workspace = isAzureUri(url) ? workspaceId : { ...extractWorkspaceDetails(workspaceStore.get().workspace) }
   return els.cell([
     url === null ?
       'Unable to generate download link.' :
-      div({ style: { display: 'flex', justifyContent: 'center' } }, [
+      div({ style: { display: 'flex', justifyContent: isAzureUri(url) ? 'left' : 'center' } }, [
         h(ButtonPrimary, {
           disabled: !url,
           onClick: () => {
             Ajax().Metrics.captureEvent(Events.workspaceDataDownload, {
-              ...extractWorkspaceDetails(workspaceStore.get().workspace),
-              fileType: _.head((/\.\w+$/).exec(uri)),
```

Comment:
I noticed that this was uri and not url, is there a reason for that or potentially a typo? I changed it to be url since that is what is set based on uri but want to make sure that is the right thing to do.